core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -721,6 +721,8 @@ class KafkaConfig private(doLog: Boolean, val props: java.util.Map[_, _], dynami
   val quorumLingerMs = getInt(QuorumConfig.QUORUM_LINGER_MS_CONFIG)
   val quorumRequestTimeoutMs = getInt(QuorumConfig.QUORUM_REQUEST_TIMEOUT_MS_CONFIG)
   val quorumRetryBackoffMs = getInt(QuorumConfig.QUORUM_RETRY_BACKOFF_MS_CONFIG)
+  private val _quorumConfig = new QuorumConfig(this)
+  def quorumConfig = _quorumConfig
```

Comment:
Could you please use this to return quorum-related configs and then remove all related getters from `KafkaConfig`? for example: `quorumElectionBackoffMs`, `quorumFetchTimeoutMs`, `quorumElectionTimeoutMs`, `quorumLingerMs` and `quorumRetryBackoffMs` can be removed. `quorumRequestTimeoutMs` can be removed and its' callers should use `QuorumConfig#requestTimeoutMs` instead